server/weaverbird/backends/sql_translator/steps/utils/query_transformation.py

```diff
@@ -185,3 +186,10 @@ def complete_fields(query: SQLQuery, columns=None) -> str:
         compiled_query = ', '.join(query_keys)
 
     return compiled_query
+
+
+def handle_zero_division(formula: str) -> str:
+    if '/' not in formula:
```

Comment:
With this logic,  we avoid Snowflake's Zero Division Error. Error values are replaced by NULL.
The user will be able to replace the NULL values by ZERO using the replace step.
Later on we can enhance this by handling operation priorities etc...
I'll document that (and other "enhancement topics") in a a dedicated page.